webapp/src/selectors/index.tsx

```diff
@@ -11,15 +11,15 @@ export const getGlobalModalState = (state: ReduxState): GlobalModalState => {
 };
 
 export const getLinkModalState = (state: ReduxState): LinkProjectModalState => {
```

Comment:
The type of state should be `GlobalState`.

Reply:
`GlobalState` does not contain our plugin state `'plugins-mattermost-plugin-azure-devops'` so we have extended the `GlobalState` in a custom interface as `ReduxState` where we have Global state along with our plugin state